src/o365/spo/commands/list/list-sitescript-get.ts

```diff
@@ -0,0 +1,225 @@
+          cmd.log(requestOptions);
+          cmd.log('');
+        }
+
+        return request.post(requestOptions);
+      })
+      .then((res: any): void => {
+        if (this.debug) {
+          cmd.log('Response:');
+          cmd.log(res);
+          cmd.log('');
+        }
+
+        const siteScript: string | null = res.value;
+        if (!siteScript) {
+          cb(new CommandError(`An error occured, the site script could not be extracted from list '${args.options.listId || args.options.listTitle}'`));
+          return;
+        }
+
+        cmd.log(siteScript);
+        cb();
+      }, (err: any): void => {
+
+        this.handleRejectedODataJsonPromise(err, cmd, cb);
+      });
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-u, --webUrl <webUrl>',
+        description: 'URL of the site where the list to extract the site script from is located'
+      },
+      {
+        option: '-l, --listId [listId]',
+        description: 'ID of the list to extract the site script from. Specify either listId or listTitle but not both'
+      },
+      {
+        option: '-t, --listTitle [listTitle]',
+        description: 'Title of the list to extract the site script from. Specify either listId or listTitle but not both'
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(): CommandValidate {
+    return (args: CommandArgs): boolean | string => {
+      if (!args.options.webUrl) {
+        return 'Required parameter webUrl missing';
+      }
+
+      const isValidSharePointUrl: boolean | string = SpoCommand.isValidSharePointUrl(args.options.webUrl);
+      if (isValidSharePointUrl !== true) {
+        return isValidSharePointUrl;
+      }
+
+      if (args.options.listId) {
+        if (!Utils.isValidGuid(args.options.listId)) {
+          return `${args.options.listId} is not a valid GUID`;
+        }
+      }
+
+      if (args.options.listId && args.options.listTitle) {
+        return 'Specify listId or listTitle, but not both';
+      }
+
+      if (!args.options.listId && !args.options.listTitle) {
+        return 'Specify listId or listTitle, one is required';
+      }
+
+      return true;
+    };
+  }
+
+  public commandHelp(args: {}, log: (help: string) => void): void {
+    const chalk = vorpal.chalk;
+    log(vorpal.find(this.name).helpInformation());
+    log(
+      `  ${chalk.yellow('Important:')} before using this command, log in to SharePoint,
+    using the ${chalk.blue(commands.LOGIN)} command.
+  
+  Remarks:
+  
+    To extract a site script from a list, you have to first log in to SharePoint
+    using the ${chalk.blue(commands.LOGIN)} command,
+    eg. ${chalk.grey(`${config.delimiter} ${commands.LOGIN} https://contoso.sharepoint.com`)}.
+        
+  Examples:
+  
+    Extract a Site Script from an existing SharePoint List with title ${chalk.grey('CustomList')}
```

Comment:
List name in the description (CustomList), doesn't match the list name in the command (ContosoList)